### ISSUE TITLE
Wrong toolbar in master branch

### ISSUE THREAD

**Issue by tagliala** [kreativgebiet/rich#162]
Hi, I'm giving a try to the master branch.

I'm using rich inside rails admin

I noticed that the toolbar is wrong (left: firefox, right: chrome)

![screen shot 2014-06-24 at 13 02 51](https://cloud.githubusercontent.com/assets/556268/3370386/754889ac-fb8f-11e3-83b6-a092a4412061.png)

I'm using the following view `app/views/rails_admin/main/_form_rich_text.html.haml`:

``` rb
= form.send 'text_area', field.method_name, field.html_attributes.reverse_merge((hdv = field.html_default_value).nil? ? {} : { :value => hdv })

= javascript_include_tag "rich/base.js"
- if form.respond_to?(:dom_id)
  :javascript
    var instance = CKEDITOR.instances['#{form.dom_id(field)}'];
    if(instance) { CKEDITOR.remove(instance); }
    CKEDITOR.replace('#{form.dom_id(field)}', #{Rich.options(field.config, field.scope_type, field.scope_id).to_json.html_safe});
- else
  :javascript
    var instance = CKEDITOR.instances['#{[form.object_name, field.name].join("_")}'];
    if(instance) { CKEDITOR.remove(instance); }
    CKEDITOR.replace('#{[form.object_name, field.name].join("_")}', #{Rich.options(field.config, [], field.scope_id).to_json.html_safe});
```

Using the gem:
![image](https://cloud.githubusercontent.com/assets/556268/3370408/ddb81174-fb8f-11e3-8595-0d1dc2aaa2e6.png)

**Comment by oktalk** [comment]
Would love a more permanent fix for this.

**Comment by torreyj** [comment]
Yes, please find a permanent fix. This issue has been a major problem for me as well. Thanks

**Comment by Nekromancer** [comment]
+1

**Comment by forsaken1** [comment]
+1

**Comment by torreyj** [comment]
Why is this still an issue for Active Admin? Can somebody please fix this. This gem rocks, but this issue is forcing me to not use it.

**Comment by kfn8dkodemonkey** [comment]
I am using Rich with Rails Admin. Having the same issue with the master branch showing the wrong toolbar. I have tried the Quick Fix - changing the icon.png.

Is there and update or another quick fix. nothing suggested seams to work.

**Comment by asecondwill** [comment]
+1

**Comment by asecondwill** [comment]
Does anybody have a workaround?  rolling back to a specific version is tricky because of this issue

https://github.com/kreativgebiet/rich/issues/179

**Comment by asecondwill** [comment]
I can get this fork: 

gem 'rich', github: 'Pixelbash/rich'

to work locally, but then it doesn't work on production.  I'm guessing more assets need adding to the precompile list  but can't figure out what.

**Comment by joeswann** [comment]
Editor of fork here - got the toolbar working on production in the end, but it still comes up with `Cannot read property 'ltr' of undefined` sporadically when deploying - only some of the time though

**Comment by Ruff9** [comment]
Same pb than @asecondwill : I cant get a workaround to work due to the issue #179 . I'm using active admin and have no clue how to make rich work.
